### DISCUSSION TITLE
Should we open up the list inclusion criteria?

### DISCUSSION THREAD

**Discussion by arcataroger** [arcataroger/awesome-engineering-games#15] [Polls]
Hey everyone, and thanks for your contributions so far! The list is only a couple of days old by this point, but already we have a few requests for games that don't fall under the existing criteria ("Very Positive" or higher on Steam).

**As a community-curated list, this should be our collective call. Do we include more games (as long as we label / categorize them appropriately), or limit it to popular, well-reviewed Steam games?**

What are your thoughts?

## Examples of games not currently included, but could be:

- New Steam releases that don't have enough reviews yet
- Older games (such as emulated DOSbox titles)
- Games not on Steam for whatever reason
- Games whose overall review score might not be Very Positive, but their recent scores are (maybe they had a poor launch, but improved it later)

## Examples of approaches we could take for adding more games:

- Lower the minimum score threshold to Positive or Mostly Positive for inclusion, but use additional symbols/emoji/etc. to highlight the Very Positive and Overwhelmingly Positive games, alongside any personal favorites from contributors
- Create a new section for "New games" that don't have enough reviews yet
- Create a section for "Older games" that aren't on Steam (and probably never will be)

**What do you think? Please answer the poll and provide any suggestions in a comment :)**

**Comment by Chad-Ballay** [comment]
I say keep it to just Steam games.  That way if nothing else it has the focus to be an amazing utility and still has plenty of items to curate.  Once the criteria becomes "everything under the sun" it becomes so huge that it's likely to have a low signal to noise ratio for usefulness. 

With that in mind, I would say tweak the criteria a bit to allow for recent course corrections in reviews.  So many games are absolute shite until some major patch gets pushed and then they magically become worth playing.

**Comment by BitTwiddleGames** [comment]
I'll start off by saying, since this is your repo @arcataroger, you should feel free to do whatever matches your vision for the repository :)

Here's my 2 cents as someone who has spent some time working on indie game development. I think it would be interesting if you were willing to include newer games that haven't received many reviews yet.

1. Steam already does a really good job of promoting games that prove popular with users. To find popular engineering games, you can usually browse by tag(programming, simulation etc) and find all the commonly discussed games.
2. It is harder to find lesser known but great games on Steam. As a large platform, it primarily relies on reviews to surface new games. There are many great games on Steam that wont receive much visibility.

If you are willing to include newer games, you have a chance to help curate and discover otherwise missed games.

**Comment by arcataroger** [comment]
Hey all, thanks for the input on this and sorry for the slow reply! Was waiting to see if more people wanted to chime in (guess not), and also Turkey Week was a bit busy :)

OK,  here is my proposal for now:

- Keep this particular list to Steam only. People are always welcome to make another sister list for non-Steam games, and I'd be happy to link to it from here.
- HOWEVER, let's expand the criteria to include:
   - Games that became Very Positive recently, even if their overall score is lower (i.e. it got better over time)
   - New games, maybe things that are < 1 year old and/or have < ~50 reviews. I agree with @BitTwiddleGames that it'd help with discovery of new engineering-focused games. I know that'd help me!

How does that sound? I'll keep this open a few more days to see if anyone has feedback, but if not, I'll go ahead and start adding more games according to those criteria.

I think at some point this will cause some edge caes to appear, but that's ok... the joy of a manually/community-curated list is that we can always deal with those on a case by case basis and exclude any games that are trying to game the system :)

Thanks again for the thoughts!
